Providers/nxOMSAutomationWorker/automationworker/worker/omsutil.py

```diff
@@ -79,6 +79,16 @@ def initialize():
     else:
         print "Successfully set owners of certificate folder."
 
+    proxy_paths = ["/etc/opt/microsoft/omsagent/conf/proxy.conf", "/etc/opt/microsoft/omsagent/proxy.conf"]
+    for path in proxy_paths:
+        if os.path.exists(path):
+            process, output, error = linuxutil.popen_communicate(["sudo", "chmod", "g+r", path])
```

Comment:
We will eventually make it in the omsadmin.sh file as well but some customer are blocked now so we have to do it here (in any case even if we make it to omsadmin not all customer update their agents so this is needed here as well). The group read permission is needed because our worker runs as a seperate user which is part of the omiusers group. Without read permission if the box has no direct access to the internet then the worker cannot work.